models/openmc/beavrs/materials.py

```diff
@@ -178,38 +178,8 @@ def openmc_materials(ppm):
 
     # Density of clean water at 2250 psia T=560F NIST
     h2o_dens = 0.73986
-
-    # Compute weight percent of natural boron in borated water
-    wB_Bh2o = ppm*1.0e-6
-
-    # Borated water density
-    rho_Bh2o = h2o_dens/(1 - wB_Bh2o)
-
-    # Compute weight percent of clean water in borated water
-    wh2o_Bh2o = 1.0 - wB_Bh2o
-
-    # Compute molecular mass of clean water
-    M_h2o = 2*atomic_weight('H') + atomic_weight('O')
-
-    # Compute molecular mass of borated water
-    M_Bh2o = 1.0/(wB_Bh2o/atomic_weight('B') + wh2o_Bh2o/M_h2o)
-
-    # Compute atom fractions of boron and water
-    aB_Bh2o = wB_Bh2o*M_Bh2o/atomic_weight('B')
-    ah2o_Bh2o = wh2o_Bh2o*M_Bh2o/M_h2o
-
-    # Compute atom fractions of hydrogen
-    ah_Bh2o = 2.0*ah2o_Bh2o
-    aho_Bh2o = ah2o_Bh2o
-
-    # Create material
-    mats['Borated Water'] = openmc.Material(name='Borated Water')
+    mats['Borated Water'] = openmc.model.borated_water(ppm, temperature=560., pressure=2250., temp_unit='F', press_unit='psi', density=h2o_dens)
```

Comment:
Let's use the default temperature here. All the material temperatures get updated in this code block:

```
for mat in mats.values():
        mat.temperature = c.operating_temperature
```

However, this is not done correctly right now (there are still some material temperatures being set to 300. Can you remove all instances of `mat.temperature = 300` in this file and move the code block above to the end of the file?

Reply:
This was addressed in the other PR that you submitted. Now you don't need to specify the operating temperature here, you can use the default